robot_smach_states/src/robot_smach_states/startup.py

```diff
@@ -19,9 +19,19 @@ class StartChallengeRobust(smach.StateMachine):
     """Initialize, wait for the door to be opened and drive inside"""
 
     def __init__(self, robot, initial_pose, use_entry_points=False, door=True):
+        """ Initialization method
```

Comment:
I like to start on a new line.

Reply:
Let's not clutter and saddle people up with what we like/not like :-)

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings, which says both are fine.

Reply:
It is also inconsistent. Most of the times, the docs start on a new line.